src/Bowl.js

```diff
@@ -0,0 +1,27 @@
+
+function Bowl(frame, roll1, roll2, score, bonus, frameTotal) {
+  this.frame = frame;
+  this.roll1 = roll1;
+  this.roll2 = roll2;
+  this.score = score;
+  this.bonus = []
+  this.frameTotal = 0
+};
+
+
+Bowl.prototype.add = function(roll1, roll2) {
+  this.score += (roll1 + roll2)
+  this.frame += 1
+  this.frameTotal += (roll1 + roll2)
```

Comment:
consider extracting frame as a separate object to adhere to the Single Responsibility Principle